### PR TITLE
added very simple bias correction hack

### PR DESCRIPTION
**Pull request by nikosbosse**
simple hack for fable_models. Adds an argument to fable_models(), all other behaviour unchanged

- bias is calculated based on fitted values
- bias is then subtracted from forecasts

not sure this is a terribly good idea, but it's worth testing

### CODE REVIEW

File: R/fable_model.R [hunk 1]
```diff
@@ -7,61 +7,77 @@
...
 #' @export
 #' @importFrom tsibble tsibble
 #' @importFrom fabletools model forecast
 #' @importFrom purrr map
-#' @importFrom dplyr bind_rows
+#' @importFrom dplyr bind_cols
 #' @examples
 #' ## Used on its own
 #' fable_model(y = EpiSoon::example_obs_rts[1:10, ]$rt,
 #'            model = fable::ARIMA(y ~ time),
```
Comment (seabbs) [inline R/fable_model.R:24]: please add as a second example - this is not likely to be the default option as it feels a bit hacky

File: R/fable_model.R [hunk 2]
```diff
@@ -7,61 +7,77 @@
...
+  forecast <- fabletools::forecast(model, h = horizon, times = samples)
 
-if (samples == 1) {
-  ## If only using a single sample use central estimate
- samples <- t(data.frame(forecast$y))
- samples <- data.frame(samples)
-}else{
-  ## Pull out distributions
-  dist <- forecast$.distribution
+  if (bias_correction == TRUE) {
```
Comment (seabbs) [inline R/fable_model.R:50]: you dont need `bias_correction == TRUE` here just `bias_correction`

File: R/fable_model.R [hunk 3]
```diff
@@ -7,61 +7,77 @@
...
+    bias_correction <- mean(residuals(model)$.resid,
+                            na.rm = T)
```
Comment (seabbs) [inline R/fable_model.R:52]: Change this to `TRUE` using `T` is bad practise as can be overwritten

File: R/fable_model.R [hunk 4]
```diff
@@ -7,61 +7,77 @@
...
-                                    sd = .$sd))
+  } else {
+    bias_correction <- 0
   }
 
-  ## Bind samples together
-  names(samples) <- 1:length(samples)
-  samples <- dplyr::bind_cols(samples)
-}
+  if (samples == 1) {
```
Comment (seabbs) [inline R/fable_model.R:58]: In general please do not copy and paste in changes as it confuses git and makes the real changes hard to find.

**Comment by nikosbosse** [comment]
not confident this is working at all. closing for now

**Pull request closed without merging**